README.md

```diff
@@ -1,13 +1,10 @@
-# SangeFault
+# <img src="https://raw.githubusercontent.com/SangeloDev/SangeFault/1.8.9/pack.png" width="32"> SangeFault
 
-[![UC2Badge](https://img.shields.io/badge/UC%C2%B2-supported-%23FE7D37)](https://uc.gamingcraft.de) [![LabymodBadge](https://img.shields.io/badge/LabyMod-supported-%23008FE8)](https://labymod.net)
-
-[![PMC](https://img.shields.io/badge/PlanetMinecraft-visit-%236EC310)](https://www.planetminecraft.com/texture-pack/sangefault-1-8-x-resource-pack/) [![Website](https://img.shields.io/badge/Coming%20Soon-pick%20and%20choose-blue)](https://pack.sangelo.space)
-
-ℹ️ If you're viewing this page from [Gitea](https://git.sangelo.space/Sangelo/SangeFault) and don't see any releases, please view them on [GitHub](https://github.com/SangeloDev/SangeFault/releases).

```

Comment:
This has to stay in, because I mirrored the project over, but am too lazy to migrate all the releases as well.

Reply:
I didn't actually remove this. Seems to be a sync issue or I accidentally did this. Will fix this.